app/views/venues/show.html.erb

```diff
@@ -4,4 +4,5 @@
   <%= link_to @venue.website, @venue.website, class: "website", target: "_blank" %>
   <p class="venue-address"><%= @venue.address %></p>
   <p class="venue-capacity">Capacity: <%= @venue.capacity %></p>
+  <p class="venue-t-accessible"><%= @venue.display_t_accessible %></p>
```

Comment:
This will display a blank line if it is not T accessible (which may throw off your styling in the future). You might want to consider having it actually return something like "No nearby T station" or something. If you don't want it to return anything, you'll need to update the view so it only shows that line if there is a (truthy) value, and you'll want the method to return nil or false